app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx

```diff
@@ -47,13 +52,41 @@ const NoteForm = ({ onClose, refetch, note, isEdit }) => {
               className="w-full flex-grow-0"
               label="Description"
               name="description"
-              rows={8}
+              rows={1}
+            />
+            <Select
+              isClearable
+              isSearchable
+              required
+              className="w-full flex-grow-0"
+              label="Assigned Contact"
+              name="assignedContact"
+              placeholder="Select contact"
+              options={CONTACTS_DATA.map(({ label, value }) => ({
+                label,
+                value,
+              }))}
```

Comment:
Same as last comment.

Reply:
I am sorry. This is a careless mistake on my end. Since the `buildSelectOptions` method is returning the `CONTACTS_DATA` in the correct format, there is no need for the `map` function here. I will change this to `options={CONTACTS_DATA}` and then, this should work.